examples/regional-backend-security-policy-example/outputs.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# output "security_policy" {
+#   value       = module.cloud_armor.policy
+#   description = "Cloud Armor security policy created"
+# }
+
+output "project_id" {
```

Comment:
If the usage of this was to get the project_id in tests, lets use https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft#TFBlueprintTest.GetTFSetupStringOutput instead. Example https://github.com/terraform-google-modules/terraform-google-service-accounts/blob/90e61434891278b8ce867338f5f5aad9dec8ab7a/test/integration/simple_sa/simple_sa_test.go#L31